qemu/tests/qemu_guest_agent.py

```diff
@@ -2547,9 +2594,19 @@ def gagent_check_fsinfo(self, test, params, env):
         fs_info_qga = self.gagent.get_fsinfo()
         for fs in fs_info_qga:
             mount_pt = fs["mountpoint"]
-            if params["os_type"] == "windows":
+            if (params["os_type"] == "windows" and
+                    mount_pt != "System Reserved"):
                 mount_pt = mount_pt[:2]
 
+            error_context.context("Check file system '%s' usage statistics." %
+                                  mount_pt, logging.info)
+            if mount_pt != 'System Reserved':
+                # disk usage statistic for System Reserved
+                # volume is not supported.
+                check_usage_qga_guest(mount_pt)
+            else:
+                logging.info("'%s' disk usage statistic is not supported" % mount_pt)
+
```

Comment:
I understand that this step is to resolve the original issue: "System reserve volume's file system via guest agent is different from it's in guest", my question is, so the solution is to give up the checking for `System Reserved` block, it won't be the same as other partition, right?
Another question is, the above check point adding about file system usage, is not to fix the issue described in the bz, right? It seems to me it's a test case updating, would you provide me the test case link in the bz so I might check the current steps. Thanks.

Reply:
1.I've confirmed System Reserved issue with developer Basil, and he told me that we don't support disk usage statistic checking for SystemReserved. just skip it is fine. 
2. yeah,it's a test case updating ,and the polarion case was added to bz,you can have a check, https://bugzilla.redhat.com/show_bug.cgi?id=1857505